python/pyspark/sql/functions.py

```diff
@@ -1028,20 +1028,28 @@ def to_timestamp(col, format=None):
 
 
 @since(1.5)
-def trunc(date, format):
+def trunc(data, format):
     """
-    Returns date truncated to the unit specified by the format.
+    Returns date truncated to the unit specified by the format or
+    number truncated by specified decimal places.
```

Comment:
in the latter case where `data` is a number, the 2nd parameter as called `format` seems a bit out of place?